File: IntroPC/Tarea 6/Rutina_primo/factorprim.cpp
#include <cstdio>
#include"rutinas_primo.cpp"
using namespace std;

int main(void)
{
  int n,i,f,j,comp,sw,p;
  printf("ingrese un numero: ");
  scanf("%d",&n);
  printf("factores primos: \n");
   for(i=1;i<=(n);i++)
    {
      
      
      comp=n%i; 
	  if (comp==0)
	    {
	     
	      p=primo(i);
	      if (p==1)
		{
		  printf(" %d \n",i);
		}
	    }

    }
}



 

File: IntroPC/Tarea 6/Rutina_primo/factorprim.py
#! /usr/bin/ env python
import rutinas_primo

n=input("ingrese un numero: ")
print"factores primos: \n"
i=1   
while i<=(n):
    comp=n%i 
    if comp==0:
        p=rutinas_primo.primo(i)
        if (p==1):
            print" %d \n"%(i)
            
    i=i+1


